### ISSUE TITLE
Midterm Report Peer Review

### ISSUE THREAD

**Issue by jt888666** [ORIE4741/ProjectsFall2019#118]
The project is about exploring whether there is any difference or bias between reviews collected by Review Collection Platform and the normal reviews listed on TripAdvisor.com or not. Before adopting the RCP platform, all reviews are normal reviews collected by organic customers. After adopting RCP platform, hotels are able to solicit reviews from all customers posted checkout. The datasets they used are reviews from hotels in New York City consisting of 430 hotels and 34853 reviews in the dataset, which excludes the hotels that are rarely adopting RCP platform. Each review contains various features, including hotel id, name of the hotel, review id, review rating, data of stay and etc. 
The project is really creative and summarizes 3 variables from text and ratings, including emotional content, objective content and extremity. For the data exploration part, the report includes a clear comparative table that divides the reviews into reviews created directly on Trip Advisor and reviews created responding to solicitation and summarizes the min, max mean and standard deviation of the data. It also includes a histogram showing the number of review increases for hotel after adopting RCP platform. Also, the report includes a table consisting of all the results from regression results.
The improvements I suggests are that they can use more complex model to achieve regularization and decrease overfitting. Also, it is helpful to employ different kinds of loss function to evaluate how well the algorithm models the datasets. Furthermore, using text embedding will enable them to analyze what kinds of words in review will influence the result.
